.github/workflows/directory_writer.yml

```
@@ -10,20 +10,24 @@ on:
   #        │  │ │ │ │
   #        │  │ │ │ │
   #        *  * * * *
-  - cron: '0 0 * * *'
-  workflow_dispatch:
+  #- cron: '0 0 * * *'
+  #workflow_dispatch:
 jobs:
   build:
-    if: github.repository == 'TheAlgorithms/Julia' # We only need this to run in our repository.
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Get Julia v1.8
+        uses: julia-actions/setup-julia@v1
         with:
-          fetch-depth: 0
-      - name: Build directory
-        uses: TheAlgorithms/scripts/directory_md@main
-        with:
-          language: Julia
-          working-directory: src
-          filetypes: jl
-          ignored-directories: docs/,pluto_notebooks/,test/
+          version: 1.8
```

The repository itself is using Julia 1.9 now.
```suggestion
          version: 1.9
```